src/zepto.js

```diff
@@ -437,10 +436,12 @@ var Zepto = (function() {
     },
 
     ready: function(callback){
-      // need to check if document.body exists for IE as that browser reports
-      // document ready when it hasn't yet created the body element
-      if (readyRE.test(document.readyState) && document.body) callback($)
-      else document.addEventListener('DOMContentLoaded', function(){ callback($) }, false)
+      // don't use "interactive" on IE <= 10 (it can fired premature)
```

Comment:
Zepto website states that IE10 supported as a primary target. IE10 has this issue. To more on this: there is pull #733 where IE9 support was accomplished in three changes (don’t counting detect module). Two of these three seems already in the master. That is third (to be more specific, fix of third from 790d6c2).